lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -1573,6 +1584,13 @@ namespace {
       ValueDecl *decl = concDeclRef.getDecl();
       AsyncMarkingResult result = AsyncMarkingResult::NotFound;
 
+      // if it is explicitly marked distributed actor independent,
+      // it is synchronously accessible, no implicit async needed.
+      if (decl->getAttrs().hasAttribute<DistributedActorIndependentAttr>() ||
+          decl->getAttrs().hasAttribute<NonisolatedAttr>()) {
```

Comment:
I still suspect we can remove `DistributedActorIndependentAttr` and settle for implicitly-introduced `NonisolatedAttr`, but we can tackle that elsewhere.

Reply:
yup! DistributedActorIndependentAttr is a bit of debt from the early impls -- nowadays nonisolated is enough.

I'll make a specific radar to follow up on the removal